processing/src/main/java/io/druid/segment/virtual/ExpressionSelectors.java

```diff
@@ -77,6 +78,12 @@ public double getDouble()
         return baseSelector.getDouble();
```

Comment:
checks?

Reply:
Expressions already have the checks, don't think we need to add redundant checks here.

Reply:
Please add comments for readers, expecting asserts

Reply:
added.